war/src/main/less/pluginSetupWizard.less

```diff
@@ -75,6 +75,12 @@
         box-sizing: border-box;
         padding: 4.2em 0 4.8em 0;
     }
+    
+    .jenkins-version {
+        float: left;
```

Comment:
Does it look wrong if it is floating right? This could be exactly the same footer as the main page footer. But that seems like a 2.1 issue at best, and not worth the keystrokes I have already spent introspecting my own ambivalence... 

   ...or does it?